backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/exportimport/ImportVmCommand.java

```diff
@@ -1177,10 +1194,19 @@ protected void updateImage(DiskImage disk) {
      * @param disk the disk being imported
      **/
     protected void saveDiskImageDynamic(DiskImage disk) {
-        DiskImageDynamic diskImageDynamic = new DiskImageDynamic();
-        diskImageDynamic.setId(disk.getImageId());
-        diskImageDynamic.setActualSize(disk.getActualSizeInBytes());
-        diskImageDynamicDao.save(diskImageDynamic);
+        DiskImageDynamic existingDiskImageDynamic = diskImageDynamicDao.get(disk.getImageId());
+
+        if (existingDiskImageDynamic != null) {
+            log.info("Disk image dynamic '{}' for shared disk '{}' already exists.",
+                    existingDiskImageDynamic.getId(), disk.getId());
+        } else {
+            DiskImageDynamic diskImageDynamic = new DiskImageDynamic();
+            diskImageDynamic.setId(disk.getImageId());
+            diskImageDynamic.setActualSize(disk.getActualSizeInBytes());
+            log.debug("Adding disk image dynamic '{}' for disk '{}' to the database.",
```

Comment:
did you check how this log looks?
there are automatic debug logs for database operations, and we never actually see debug logs except for OST

Reply:
Here is a log example:
```
2022-03-27 16:41:43,548+03 INFO  [org.ovirt.engine.core.bll.exportimport.ImportVmCommand] (EE-ManagedThreadFactory-engine-Thread-3212) [1e75e677-39e3-4930-bbad-4777eed7c3c2] Adding disk image dynamic 'a83da3d3-6938-4058-9b43-192ec47438c3' for disk 'af724768-041c-4201-bd64-eb59d7940561' to the database.
```
Initially the logs were with `info` level and Arik suggested to change them to `debug` or totally remove.
I found it helpful to have those logs during development and thought that it will be also helpful during debugging real problems should they happen.
That's why I left the logs, but with `debug` level, while logs describing "not adding existing shared disks" have `info` level.
If you think that they should be removed (all / only those with `debug` level) I'll do it.